api/app/signals/apps/questionnaires/tests/test_models.py

```diff
@@ -370,3 +374,47 @@ def test_pathological_case_key_is_valid_uuid(self):
 
         retrieved = Question.objects.get_by_reference(str(generated))
         self.assertEqual(retrieved, question)
+
+
+class TestQuestion(TestCase):
```

Comment:
Shouldn't this be called something like `TestSession`?

Reply:
agree, will fix

Reply:
👍🏼